Sources/Widgets/Widgets3D/ResliceCursorWidget/behavior.js

```diff
@@ -68,6 +69,42 @@ export default function widgetBehavior(publicAPI, model) {
     model.widgetState.deactivate();
   };
 
+  publicAPI.handleStartMouseWheel = (callData) => {
+    publicAPI.handleMouseWheel(callData);
+  };
+
+  publicAPI.handleMouseWheel = (calldata) => {
+    const direction = calldata.spinY;
+    const currentCenter = model.widgetState.getCenter();
+    const imageSpacing = model.widgetState.getImage().getSpacing();
+    const imageBounds = model.widgetState.getImage().getBounds();
+    const bounds = vtkBoundingBox.newInstance({ bounds: imageBounds });
+
+    // Get the normal use to translate the center along
+    const normal = calldata.pokedRenderer
+      .getRenderWindow()
+      .getRenderers()[0]
+      .getActiveCamera()
+      .getDirectionOfProjection();
+
+    // Direction of the projection if the inverse of what we want
+    const correctNormal = vtkMath.multiplyScalar(normal, -1);
+
+    const newCenter = [
+      currentCenter[0] + direction * correctNormal[0] * imageSpacing[0],
+      currentCenter[1] + direction * correctNormal[1] * imageSpacing[1],
+      currentCenter[2] + direction * correctNormal[2] * imageSpacing[2],
+    ];
+
+    // Don't allow to move out of the image
+    if (bounds.containsPoint(...newCenter)) {
```

Comment:
that may prevent you from going on the edge of the bounds :-/

Reply:
if you think that it is not a requirement, I can remove it.

Reply:
This is a requirement :-) Sorry I was not being clear.

I meant you should project new center onto the edge if it is outside (you can ask Paul for what he did on some other project)

Reply:
I didn't see the last comment. That's why I didn't understand your last comment..